packages/flutter_tools/test/integration.shard/test_driver.dart

```diff
@@ -152,13 +152,13 @@ abstract class FlutterTestDriver {
     if (_processPid == null)
       return -1;
     _debugPrint('Sending SIGTERM to $_processPid..');
-    Process.killPid(_processPid);
+    ProcessSignal.SIGTERM.send(_processPid);
```

Comment:
Could these be what's breaking Windows?

Reply:
These were actually to fix Windows. But it's definitely possible...